frontends/web/src/locales/en/app.json

```diff
@@ -300,7 +300,7 @@
       },
       "mobile": {
         "false": "Closed",
-        "label": "Mobile App",
+        "label": "Mobile app",
```

Comment:
I'm not sure where this is used but maybe we should more consistently start using "mobile verification app" (or something like this) in preparation for the Android release of this app.

Reply:
won't do now as i'd have to check if it fits the UI